config/eeid-params.conf

```diff
@@ -0,0 +1,14 @@
+# OpenEnclave (EEID) settings for SGX-LKL:
```

Comment:
Depends on who the "user" is. I think we should get rid of this file altogether now. Someone who wants to start a container wants to edit heap/stack sizes, which they can do in the enclave config. Someone who wants to build/extend SGX-LKL wants to edit the debug flag (which we can `#ifdef`). Someone who wants to publish a container wants to edit security/product version numbers, which they would have to change in the source file.

Reply:
We don't want to use security/product fields.
Let's open a separate issue about removing the conf file, I suspect this will take a bit longer to fully discuss.

Reply:
Yeah, that's a good call. BTW, we can also support both, I think by arbitrarily made up OE rules, a file overrides the macro.

Reply:
https://github.com/lsds/sgx-lkl/issues/574